libnmstate/nm/plugin.py

```diff
@@ -136,6 +137,7 @@ def get_interfaces(self):
             iface_info.update(nm_user.get_info(self.context, dev))
             iface_info.update(nm_lldp.get_info(self.client, dev))
             iface_info.update(nm_team.get_info(dev))
+            iface_info.update(get_infiniband_info(applied_config))
```

Comment:
Do we want to import the functions now? If so, I would refactor the others `nm_...get_info()` in another PR, I also prefer it.

Reply:
I prefer function over import the whole module.